layouts/shortcodes/docs/instrumentation/manual-intro.md

```diff
@@ -0,0 +1,16 @@
+[Manual instrumentation](/docs/concepts/instrumentation/manual/) is the act
+of adding observability code to your application yourself.
+
+If you're instrumenting an application, you need to use the OpenTelemetry SDK
```

Comment:
instrumenting part just requires API only.

Reply:
True, but...it serves no purpose without an exporter, which you need the SDK for.

Reply:
@reyang what part is not necessarily true? That it serves no purpose without an exporter or that you need the SDK, can you elaborate?